care/facility/summarisation/tests_summary.py

```diff
@@ -0,0 +1,104 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.conf import settings
+from django.utils import timezone
+
+from celery.decorators import periodic_task
```

Comment:
Codacy found an issue: [Unable to import 'celery.decorators'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5838242)